### PR TITLE
Add Sass to project. Related #79

### LINKED ISSUE #63: Clean up main.css
remove comment here: https://github.com/CYPIAPT-LNDSE/welcome-to-camhs/blob/master/main.css#L265

### LINKED ISSUE #79: Add Sass to project
Add Sass to project and refactor current CSS in Sass. 

We have decided to not use Tachyons with Sass. We did some research into both and it seems that the best was to write custom styles in Tachyons is in the Tachyeon's style sheet rather than creating a separate style sheet (in CSS or Sass) and using them both together.

### PR DESCRIPTION
**Pull request by RhodesPeter**
- Add Sass to project, Fixes #79 
- Transpose original CSS into Sass
- Delete original main.css file
- Fixes #63, clean up main.css

### CODE REVIEW

File: public/main.css.map [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
```
Comment (des-des) [inline public/main.css.map]: This is compiled? Can we find a way not to commit this!

File: scss/_eating.scss [hunk 2]
```diff
@@ -0,0 +1,99 @@
+.eating {
+  @extend %main-content;
+}
+
+.eating__text {
```
Comment (des-des) [inline scss/_eating.scss]: This is also defined in main.css

File: scss/_choose-an-avatar.scss [hunk 3]
```diff
@@ -0,0 +1,29 @@
+.avatar {
+  @extend %main-content;
+  padding: 0;
+}
+
+.avatar__text {
+  @extend %main-text;
+}
+
+.avatar__title {
```
Comment (des-des) [inline scss/_choose-an-avatar.scss]: Ok this is also second definition.

File: scss/_eating.scss [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+.eating__slider {
+  width: 600px;
+  position: relative;
+  overflow-y: hidden;
+  height: 540px;
+  margin-bottom: 60px;
+}
+
+.slider__tree {
```
Comment (des-des) [inline scss/_eating.scss]: And here

**Comment by skibinska** [comment]
hi @des-des , we have added main.css to .gitignore but  we don't understand the other comments.

**Comment by des-des** [comment]
@skibinska I made the other comments before I realised you had commited your compiled css, ignore them.

image
image

This is still commited. You need to give me a pull request where this is not in the diff.

**Comment by RhodesPeter** [comment]
Ok we have removed the compiled main.css file @des-des.

**Comment by RhodesPeter** [comment]
Hi @des-des. I have created some new variables in the _variables.scss file.

**Pull request merged**